app/src/main/java/org/sopt/sample/presentation/ProfileViewModel.kt

```diff
@@ -0,0 +1,24 @@
+package org.sopt.sample.presentation
+
+import androidx.lifecycle.ViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import org.sopt.sample.presentation.model.UserInfo
+import org.sopt.sample.util.InSoptSharedPreference
+import javax.inject.Inject
+
+@HiltViewModel
+class ProfileViewModel @Inject constructor(val inSoptSharedPreference: InSoptSharedPreference) :
```

Comment:
private 접근자 빼먹은 것 같네용!
```suggestion
class ProfileViewModel @Inject constructor(private val inSoptSharedPreference: InSoptSharedPreference) :
```